lib/flop.ex

```diff
@@ -2004,13 +2027,31 @@ defmodule Flop do
   defp get_order_direction(directions, index),
     do: Enum.at(directions, index, :asc)
 
-  defp new_order_direction(0, :asc), do: :desc
-  defp new_order_direction(0, :asc_nulls_first), do: :desc_nulls_last
-  defp new_order_direction(0, :asc_nulls_last), do: :desc_nulls_first
-  defp new_order_direction(0, :desc), do: :asc
-  defp new_order_direction(0, :desc_nulls_first), do: :asc_nulls_last
-  defp new_order_direction(0, :desc_nulls_last), do: :asc_nulls_first
```

Comment:
I pulled these function heads out into the map to make their relationship more explicit, given the additional function heads added (see below).

Reply:
If you change the default as described in the other comment, you don't need additional function heads. And then I definitely prefer the previous version without the module attribute.